goc/models.py

```diff
@@ -29,7 +33,7 @@ def __repr__(self):
 class Author(db.Model):
     id = db.Column(db.Integer, primary_key=True)
     name = db.Column(db.String(20), nullable=False)
-    blogs = db.relationship('Blog', backref='author', lazy=True)
+    blogs = db.relationship('Blog', backref='authorr', lazy=True)
```

Comment:
Is this intended?

Reply:
Without this errors are coming .The names under backref and the attribute author in blog defining a relationship need to be different